### PR TITLE
Added helper function for symlink cleanup

### PR DESCRIPTION
**Pull request by priyanka19-98**
This function will be called to clean up the symlinks created by PVs once the LV/LVSets are deleted.

Signed off by: Priyanka Jiandani <pjiandan@redhat.com>

### CODE REVIEW

**Comment by priyanka19-98** [comment]
@rohan47

File: common/symlink_utils.go [hunk 1]
```diff
@@ -37,3 +42,23 @@ func GetSymLinkSourceAndTarget(dev internal.BlockDevice, symlinkDir string) (str
...
+
+func GetCleanPVSymlinkFunc(runtimeConfig *provCommon.RuntimeConfig) func(pv *v1.PersistentVolume) error {
+	return func(pv *v1.PersistentVolume) error {
+		log.Infof("Removing symlink for %s", pv.ObjectMeta.Name)
+		config, ok := runtimeConfig.DiscoveryMap[pv.Spec.StorageClassName]
+		if !ok {
+			return fmt.Errorf("Unknown storage class name %s", pv.Spec.StorageClassName)
+		}
+		mountPath, err := provCommon.GetContainerPath(pv, config)
+		if err != nil {
```
Comment (sp98) [inline common/symlink_utils.go:54]: this `err` is not returned.

**Review by sp98 — CHANGES_REQUESTED** [review]
Please add more details to the ticket description and commit message. Like why this change is needed in the first place, etc.

File: common/symlink_utils.go (hunk 1 above)
Comment (priyanka19-98) [reply common/symlink_utils.go:54]: Thanks, made the change.

**Pull request closed without merging**